funnel/templates/event.html

```diff
@@ -62,9 +62,9 @@
             </li>
             <li>
             {%- if p.checked_in %}
-              <a href="{{ event.name }}/checkin/{{ p.id }}" class='btn btn-xs btn-danger'>Cancel Check-in</a>
+              <a href="{{ event.name }}/checkin/{{ p.id }}" data-pid="{{ p.id }}" data-tocheckin=false class='btn btn-xs cancel-checkin'>Cancel Check-in</a><img src="{{ url_for('static', filename='img/ajax-loader.gif')}}" class="loader"/>
```

Comment:
In `data-tocheckin=false`, attribute values should always be quoted as unquoted values are an attack vector. So `data-tocheckin="false"`.